src/Network/Ricochet/Connection.hs

```diff
@@ -24,53 +25,88 @@ import           Network.Socks5           (socksConnectTo)
 import           System.IO                (BufferMode (..), Handle (),
                                            hSetBuffering)
 
+-- | Creates a new RicochetState listening on the supplied port
 createState :: PortID -> IO RicochetState
 createState port = do
   sock <- listenOn port
   return $ MkRicochetState sock [] [] (PortNumber 9050) M.empty
 
+-- | Waits until a new peer connects to initiae a connection
 awaitConnection :: Ricochet Connection
 awaitConnection = do
   sock <- use serverSocket
   (handle, _, _) <- liftIO $ accept sock
-  initConnection handle False
+  initiateConnection handle False
 
-connectTo :: String -> PortID -> Ricochet Connection
+-- | Connects to a peer through the Tor network
+connectTo :: String              -- ^ Tor hidden service identifier to connect to
+          -> PortID              -- ^ Port to connect to
+          -> Ricochet Connection -- ^ Returns the established connection
 connectTo domain port = do
   torPort <- use socksPort
   handle <- liftIO $ socksConnectTo "localhost" torPort domain port
-  initConnection handle True
+  initiateConnection handle False
 
-initConnection :: Handle -> Bool -> Ricochet Connection
-initConnection handle isClientSide = do
+-- | Initiate a newly made connection
+initiateConnection :: Handle              -- ^ Handle corresponding to the connection to the peer
+                   -> Bool                -- ^ Wether this side accepted the connection
```

Comment:
This is exactly the opposite of what `isClientSide` says.  It should be “Whether this side opened the connection.”